app.rb

```diff
@@ -1,9 +1,22 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/peeps'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+post '/peeps' do 
+  Peeps.create(message: params[:message], date: params[:date])
+  redirect '/peeps'
+end 
```

Comment:
Take care to keep your indentation consistent. Linters like rubocop can help with that.